rocky/tests/katalogus/test_katalogus.py

```diff
@@ -1,51 +1,151 @@
 import pytest
 from django.core.exceptions import PermissionDenied
 from katalogus.client import KATalogusClientV1, parse_plugin
-from katalogus.views.katalogus import KATalogusView
+from katalogus.views.katalogus import AboutPluginsView, BoefjeListView, KATalogusView, NormalizerListView
 from katalogus.views.katalogus_settings import ConfirmCloneSettingsView, KATalogusSettingsView
 from pytest_django.asserts import assertContains, assertNotContains
 
 from rocky.health import ServiceHealth
-from tests.conftest import create_member, get_boefjes_data, setup_request
+from tests.conftest import create_member, get_boefjes_data, get_normalizers_data, get_plugins_data, setup_request
 
 
-def katalogus_plugin_listing(request, member, rf, mocker):
+@pytest.mark.parametrize("member", ["superuser_member", "admin_member", "redteam_member", "client_member"])
+def katalogus_plugin_listing(request, rf, member, mocker):
+    plugins = get_plugins_data()
     mock_requests = mocker.patch("katalogus.client.requests")
     mock_response = mocker.MagicMock()
     mock_requests.Session().get.return_value = mock_response
-    mock_response.json.return_value = get_boefjes_data()
-
+    mock_response.json.return_value = plugins
     member = request.getfixturevalue(member)
 
-    return KATalogusView.as_view()(
+    response = KATalogusView.as_view()(
         setup_request(rf.get("katalogus"), member.user), organization_code=member.organization.code
     )
+    assert response.status_code == 200
+    assertContains(response, "KAT-alogus")
+    assertContains(response, "An overview of all available plugins.")
+    # active toolbar
+    assertContains(
+        response, '<li aria-current="page"><a href="/en/' + member.organization.code + '/kat-alogus/">All</a></li>'
+    )
+    assertContains(response, '<li><a href="/en/' + member.organization.code + '/kat-alogus/">Boefjes</a></li>')
+    assertContains(response, '<li><a href="/en/' + member.organization.code + '/kat-alogus/">Normalizers</a></li>')
+    assertContains(response, '<li><a href="/en/' + member.organization.code + '/kat-alogus/">About plugins</a></li>')
+    assertContains(response, "All plugins")
+    assertContains(response, str(len(plugins)) + " Plugins available")
+    assertContains(response, "Filter plugins")
+    assertContains(response, "Clear filter")
+    assertContains(response, "filter_options")
+    assertContains(response, "sorting_options")
```

Comment:
These checks are not relevant as they're rendered regardless of view state or plugin type chosen.